### PR TITLE
AioHttpIntegration sentry_app_handle() now ignores ConnectionResetError

### PR DESCRIPTION
**Pull request by cmalek**
This just means that the upstream closed the connection.  aiohttp itself just debug logs this and discards it, so sentry_sdk should, too.

### CODE REVIEW

File: sentry_sdk/integrations/aiohttp.py [hunk 1]
```diff
@@ -115,6 +115,9 @@ async def sentry_app_handle(self, request, *args, **kwargs):
                     except asyncio.CancelledError:
                         transaction.set_status("cancelled")
                         raise
+                    except ConnectionResetError:
+                        transaction.set_status("connection_reset")
```
Comment (sl0thentr0py) [inline sentry_sdk/integrations/aiohttp.py:119]: the span status needs to be one in the list defined [here](https://develop.sentry.dev/sdk/event-payloads/span/#attributes)

**Review by sl0thentr0py — CHANGES_REQUESTED** [review]

File: sentry_sdk/integrations/aiohttp.py (hunk 1 above)
Comment (cmalek) [reply sentry_sdk/integrations/aiohttp.py]: Thanks for reviewing my pull request!  I've updated the patch to consider `ConnectionResetError` to have "cancelled" status.

**Comment by cmalek** [comment]
I did make the changes you requested, @sl0thentr0py.

**Review by sl0thentr0py — APPROVED** [review]

**Comment by sl0thentr0py** [comment]
thx for the contribution @cmalek!

**Pull request merged**